common/network/iota_packet.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#ifndef __COMMON_NETWORK_IOTA_PACKET_H__
+#define __COMMON_NETWORK_IOTA_PACKET_H__
+
+#include "common/network/socket.h"
+#include "common/trinary/tryte.h"
+
+#define TRANSACTION_PACKET_SIZE 1650
+
+typedef struct {
+  tryte_t data[TRANSACTION_PACKET_SIZE];
+  socket_addr_t source;
```

Comment:
`socket_addr_t` is POSIX as well

Reply:
That's actually a type I declare in network/socket.h